content/doc/book/security/scripted.adoc

```diff
@@ -0,0 +1,70 @@
+---
+title: Security for Scripts
+layout: section
+---
+ifdef::backend-html5[]
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+Apache Groovy scripts can be executed on the Jenkins controller and agents through various mechanisms including:
+
+* link:/doc/book/managing/script-console/[Script Console]
+* link:https://plugins.jenkins.io/groovy/[Groovy plugin] 
+when using the "Execute system Groovy script" step
+* link:https://plugins.jenkins.io/email-ext/[Extended Email plugin]
+* link:https://plugins.jenkins.io/groovy/[Job DSL plugin]
+* link:https://www.jenkins.io/doc/book/pipeline/[Pipelines]
+
+Scripts are powerful and useful tools but they must be managed carefully
+to ensure that they are not used to compromise your installation:
+
+* Limit the people to whom the *Overall/Administer* permission is granted.
+This allows people to use the
+link:/doc/book/managing/script-console/[Script Console].
+This authorization allows one to type in and execute an Apache Groovy script
+in a largely unrestricted environment
+so it should only be available to highly trusted, sophisticated users.
+
+
+Two facilities are provided that allow users with lesser permissions
+(such as *Job/Configure* or *Job/Build*) to run scripts that may be associated with plugins or their applications:
+
+* Jenkins maintains a list of "approved" scripts that cannot do any damage.
+If a user attempts to run a script that is not approved,
+the script is blocked until an administrator reviews the script and approves it
+using the _Manage Jenkins » In-process Script Approval_ screen.
+See the link:/doc/book/managing/script-approval/[In-process Script Approval] documentation for more information.
+In most cases, you should use the
+link:/doc/book/managing/script-approval/#approve-assuming-permissions-check[Approve assuming permissions check] option rather than the simple Approve option.
```

Comment:
```suggestion
```

No this is not true at all and in fact that option should rarely if ever be used.

Anyway this option pertains to sandbox signatures, unrelated to whole-script approval which this bullet point is about.